.github/workflows/publish.yml

```
@@ -68,6 +68,7 @@ jobs:
         with:
           name: web_map_editor
           path: dist/
+          ref: main
```

actionlint (1.7.9)

input "ref" is not defined in action "actions/upload-artifact@v4". available inputs are "compression-level", "if-no-files-found", "include-hidden-files", "name", "overwrite", "path", "retention-days"

(action)